cmd/init-repo/main.go

```diff
@@ -60,8 +54,6 @@ func main() {
 	repoOrg := flag.String("source-org", "", "the name of the source repository organization, (eg. kubernetes)")
 	rulesFile := flag.String("rules-file", "", "the file with repository rules")
 	targetOrg := flag.String("target-org", "", `the target organization to publish into (e.g. "k8s-publishing-bot")`)
-	skipGodep := flag.Bool("skip-godep", false, `skip godeps installation and godeps-restore`)
```

Comment:
Just as a note...

This flag is only used for `init-repo`. For specifying if godeps should be generated or not, `skip-godeps` is used in the rules file:

https://github.com/kubernetes/publishing-bot/blob/4437c32dbde906dd4a9d7ab2ca0cea6b72dc6146/cmd/publishing-bot/config/rules.go#L69

https://github.com/kubernetes/publishing-bot/blob/b1513c6d5c7dbbec57fbc1c3f3e339c24e16aba7/cmd/publishing-bot/publisher.go#L291-L293

This PR only removes the `skip-godeps` flag from `init-repo`. It does *not* remove the `skip-godeps` option from rules.